installer/scripts/omsadmin.sh

```diff
@@ -349,6 +349,21 @@ onboard_scom()
     make_dir $CONF_DIR/omsagent.d
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
+    save_config
```

Comment:
Why did you add a save_config call to both onboard_scom and onboard_lad? That method writes to `omsadmin.conf`, which is not an appropriately named file for you to store your variables.